Sample/SampleForageSDK/Sections/CardNumber/CardNumberView.swift

```diff
@@ -45,12 +45,42 @@ class CardNumberView: UIView {
         return tf
     }()
     
-    private let statusLabel: UILabel = {
+    private let firstResponderLabel: UILabel = {
         let label = UILabel()
-        label.text = "Enter Card Number"
+        label.text = "isFirstResponder: false"
         label.textColor = .gray
         label.font = UIFont.systemFont(ofSize: 14, weight: .bold)
-        label.accessibilityIdentifier = "lbl_status"
+        label.accessibilityIdentifier = "lbl_first_responder"
+        label.isAccessibilityElement = true
+        return label
+    }()
+    
+    private let completeLabel: UILabel = {
+        let label = UILabel()
+        label.text = "isComplete: false"
+        label.textColor = .gray
+        label.font = UIFont.systemFont(ofSize: 14, weight: .bold)
+        label.accessibilityIdentifier = "lbl_complete"
+        label.isAccessibilityElement = true
+        return label
+    }()
+    
+    private let emptyLabel: UILabel = {
+        let label = UILabel()
+        label.text = "isEmpty: true"
+        label.textColor = .gray
+        label.font = UIFont.systemFont(ofSize: 14, weight: .bold)
+        label.accessibilityIdentifier = "lbl_empty"
+        label.isAccessibilityElement = true
+        return label
+    }()
+    
+    private let validLabel: UILabel = {
+        let label = UILabel()
+        label.text = "isValid: true"
```

Comment:
should we show `panNumberTextField.isValid` instead of these hardcoded default values?

This will help us ensure the default/empty state values are set correctly

Reply:
[e1274be](https://github.com/teamforage/forage-ios-sdk/pull/89/commits/e1274be522afe47ea597d885a4a538ca54b8caaa)